src/studio-home/tabs-section/courses-tab/courses-filters/index.jsx

```diff
@@ -95,7 +95,7 @@ const CoursesFilters = ({
 
   const handleSearchCoursesDebounced = useCallback(
     debounce((value) => handleSearchCourses(value), 400),
-    [],
+    [activeOnly, archivedOnly],
```

Comment:
Just one qestion: should `order` not be included as well ?

Reply:
Also, going back to [this fix](https://github.com/openedx/frontend-app-authoring/pull/1307/files#diff-4a1604cf9eacd47775a682b54470137ba5367384f8923e983c6673150efe0781R44) that addressed [this issue](https://github.com/openedx/frontend-app-authoring/pull/1307#issuecomment-2362508219) in particular, after searching even if the sorting is set to "newest" or "oldest" the results are returned sorted only by display name. Could that be fixed by adding order here?

Reply:
@bradenmacdonald @mariajgrimaldi: thanks for your feedback. I already updated the branch

please let me know if anything else